Makefile

```
@@ -228,8 +247,8 @@ helm-smoke-kind:
 	KUBE_CONTEXT=kind-codex-lb-smoke IMAGE_REGISTRY=ghcr.io IMAGE_REPOSITORY=soju06/codex-lb IMAGE_TAG=ci ./scripts/helm-kind-smoke.sh external-db
 
 .PHONY: ci-fast ci
-ci-fast: lint typecheck frontend-test test-unit package
+ci-fast: lint typecheck rust-check frontend-test test-unit package
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)